app/src/main/java/de/westnordost/streetcomplete/quests/air_pump/AddBicyclePump.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.air_pump
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddBicyclePump : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes, ways with
+         amenity = bicycle_repair_station or shop = bicycle
```

Comment:
For shop=bicycle [OSM Wiki](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dbicycle#Additional_keys) claims: 

> If a bicycle shop is not tagged with a key from the above list, it means that its status is unknown

so it should be fine. Maybe linking https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dbicycle&oldid=2305390#How_to_map in a comment would be a good idea?

Note that https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbicycle_repair_station takes no position on that, is `amenity=bicycle_repair_station service:bicycle:pump=yes` implying that it is sole service? I asked on https://wiki.openstreetmap.org/wiki/Talk:Tag:amenity%3Dbicycle_repair_station#Is_presence_of_one_service_tag_implying_no_for_other.3F

Reply:
Thanks, I've documented in comment the meaning of undefined `service:bicycle:pump`.

Also I'll reply on wiki there shortly; in my experience, the whole point of introduction `X:Y=yes/no` instead of `X=Y1;Y2` was mostly just to fix that uncertainty, e.g.:
 
- using `service:bicycle:pump=yes`+`service:bicycle:rental=no` **only** defines that:
  -  yes, bicycle pump exists here
  -  no, bicycles are not rented here
  -  and says **nothing** about any and all other possible services (there are explicitly unknown)
- while traditional (example tagging) `service=bicycle_pump` was very ambiguous -  it did say that bicycle pump exists there; but it was not marked that:
  - it is known that bicycles are not being rented there
  - is the lack of other values (like "are bicycles washed here for a fee?") simply because there have not been surveyed, or because that they are known to not exist.